spring-boot/ServletReg/src/main/java/com/baeldung/props/PropertySourcesLoader.java

```diff
@@ -0,0 +1,20 @@
+package com.baeldung.props;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.ConfigurableEnvironment;
+
+@Configuration @ComponentScan(basePackages = { "com.baeldung.*" }) @PropertySource("classpath:custom.properties") public class PropertySourcesLoader {
```

Comment:
put each annotation on a separate line, and the class declaration on its own line too